### PR TITLE
Update Configuration-Settings, NFS and fix all broken links

### PR DESCRIPTION
**Pull request by ZacBlanco**
- Cleaned up `Configuration-Settings.md`
- Cleaned up `NFS.md`
- Fixed all broken links

### CODE REVIEW

**Comment by ZacBlanco** [comment]
@ns1123 PTAL :)

File: docs/en/basic/Configuration-Settings.md [hunk 1]
```diff
@@ -154,12 +155,12 @@ An Alluxio property can be possibly configured in multiple sources.
 In this case, its final value is determined by the following priority list, from highest priority to lowest:
 
 1. [JVM system properties (i.e., `-Dproperty=key`)](http://docs.oracle.com/javase/jndi/tutorial/beyond/env/source.html#SYS)
-2. [Environment variables](#use-environment-variables)
-3. [Property files](#use-site-property-files-recommended):
+2. [Environment variables](#environment-variables)
+3. [Property files](#alluxio-siteproperties-files-recommended):
 When an Alluxio cluster starts, each server process including master and worker searches for
 `alluxio-site.properties` within the following directories in the given order, stopping when a match is found:
 `${HOME}/.alluxio/`, `/etc/alluxio/`, and `${ALLUXIO_HOME}/conf`
```
Comment (ns1123) [inline docs/en/basic/Configuration-Settings.md]: I think we search the current working directory as well? Isn't that why using --files and passing alluxio-site.properties in spark works for setting client properties? @calvinjia

File: docs/en/ufs/Minio.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ t; Request ID: 158681CA87E59BA0; S3 Extended Request ID: 2d47b54e-7dd4-4e32-bc6e
 ### DNS Resolution - Unable to execute HTTP request
 
 If an exception like this is encountered then it may be that the alluxio property
-[`alluxio.underfs.s3.disable.dns.bucket`]({{ '/en/reference/Properties-List.html' | relativize_url}}#alluxio.underfs.s3.disable.dns.bucket)
+[`alluxio.underfs.s3.disable.dns.buckets`]({{ '/en/reference/Properties-List.html' | relativize_url}}#alluxio.underfs.s3.disable.dns.buckets)
 is set to `false`.
 Setting this value to `true` for Minio will allow alluxio to resolve the proper bucket location.
```
Comment (ns1123) [inline docs/en/ufs/Minio.md]: Capitalize Alluxio?

File: docs/en/ufs/NFS.md [hunk 3]
```diff
@@ -54,8 +58,8 @@ Run a simple example program:
 ./bin/alluxio runTests
 ```
 
-Visit your NFS volume to verify the files and directories created
-by Alluxio exist. For this test, you should see files named like:
+Visit your NFS volume at `/mtn/nfs` to verify the files and directories created by Alluxio exist.
```
Comment (ns1123) [inline docs/en/ufs/NFS.md]: typo. Fix to '/mnt/nfs'

File: docs/en/ufs/NFS.md [hunk 4]
```diff
@@ -54,8 +58,8 @@ Run a simple example program:
...
+For this test, you should see files named like:
```
Comment (ns1123) [inline docs/en/ufs/NFS.md]: Remove the like. Or change to "file with the naming convention". Might just be me, but "like" seems too informal for documentation.

File: docs/en/ufs/Qiniu-KODO.md [hunk 5]
```diff
@@ -58,7 +58,7 @@ according to this [order](https://mars-assets.qnssl.com/alluxio_host.png)
 ### Nested Mount
 An Kodo location can be mounted at a nested directory in the Alluxio namespace to have unified
 access to multiple under storage systems. Alluxio's
-[Mount Command](Command-Line-Interface.html#mount) can be used for this purpose.
+[Mount Command]({{ 'en/Basic/Command-Line-Interface.html' | relativize_url}}#mount) can be used for this purpose.
```
Comment (ns1123) [inline docs/en/ufs/Qiniu-KODO.md]: Should 'Mount Command' be lower case 'mount command'. The sentence currently looks like, 'Alluxio Mount Command can be used for this purpose.'

File: docs/en/ufs/NFS.md (hunk 4 above)
Comment (ZacBlanco) [reply docs/en/ufs/NFS.md]: agreed

File: docs/en/basic/Configuration-Settings.md (hunk 1 above)
Comment (calvinjia) [reply docs/en/basic/Configuration-Settings.md]: I think we also search on the classpath

**Comment by calvinjia** [comment]
merged as part of #9318

**Pull request closed without merging**